### DISCUSSION TITLE
Final Solution Deadline (Extension)

### DISCUSSION THREAD

**Discussion by afaroo01** [larics/icuas22_competition#54] [General]
Dear Organizers, 

Owing to the current Easter break, our institute/lab is closed and we as a team couldn't work during this time. We would like to request to consider to allow a little extension to submit final solution, possibly until May 8th (one week more), off course taking into consent of other teams as well. I hope this will allow us and most of the teams working towards their solutions to complete had left unattended to the public holidays.

Thanks

**Comment by fpetric** [comment]
Hi @afaroo01, 

we are looking into it. Stay tuned for an announcement here.
